### DISCUSSION TITLE
Reequired the tensorfow file for db_resnet 50 modeel file

### DISCUSSION THREAD

**Discussion by Shiva-Ai4m** [mindee/doctr#1223] [General]
I require db_resnet50 model file for other purpose please can anyone help in this @felixT2K

**Comment by felixT2K** [comment]
@Shivashankarar you can download it from 
pytorch: `https://doctr-static.mindee.com/models?id=v0.3.1/db_resnet50-ac60cadc.pt&src=0`
tensorflow: `https://doctr-static.mindee.com/models?id=v0.2.0/db_resnet50-adcafc63.zip&src=0`

**Reply by Shiva-Ai4m** [reply]
can i get a .pb file  for tensorflow @felixT2K
